bot/code_review_bot/backend.py

```diff
@@ -83,10 +83,32 @@ def publish_issues(self, issues, revision):
             return
 
         assert revision.issues_url is not None, "Missing issues_url on revision"
+        published = 0
         for issue in issues:
-            issue.on_backend = self.create(revision.issues_url, issue.as_dict())
+            payload = issue.as_dict()
```

Comment:
Nit: can we store the hash as an attribute of issue so we don't have to convert the issue to dict?

Reply:
When would we build the hash ?